*** tests/A-01-15-M-ok.out	2024-10-11 16:08:09.350580000 +0100
--- tests/A-01-15-M-ok.outhyp	2024-10-11 21:25:38.494789982 +0100
***************
*** 17,24 ****
  4 - Retirar responsabilidade
  5 - Calcular satisfação de funcionário
  0 - Sair
! Escolha uma opção: TRT|W24|Rohit Figueiredo
! Gestão de Funcionários
  1 - Visualizar todos os funcionários
  2 - Registar funcionário
  3 - Atribuir responsabilidade
--- 17,23 ----
  4 - Retirar responsabilidade
  5 - Calcular satisfação de funcionário
  0 - Sair
! Escolha uma opção: Gestão de Funcionários
  1 - Visualizar todos os funcionários
  2 - Registar funcionário
  3 - Atribuir responsabilidade
