*** tests/A-01-13-M-ok.out	2024-10-11 16:08:09.350580000 +0100
--- tests/A-01-13-M-ok.outhyp	2024-10-11 21:25:38.275798446 +0100
***************
*** 16,27 ****
  3 - Transferir para habitat
  4 - Calcular satisfação de animal
  0 - Sair
! Escolha uma opção: ANIMAL|A1|Farrusco 1|E1|VOID|H1
! ANIMAL|A2|Farrusco 2|E2|VOID|H1
! ANIMAL|A3|Farrusco 3|E3|VOID|H2
! ANIMAL|B1|Farrusco 4|E3|VOID|H2
! ANIMAL|X1|Farrusco 5|E2|VOID|H3
! ANIMAL|X2|Farrusco 6|E1|VOID|H3
  Gestão de Animais
  1 - Visualizar todos os animais
  2 - Registar animal
--- 16,27 ----
  3 - Transferir para habitat
  4 - Calcular satisfação de animal
  0 - Sair
! Escolha uma opção: hva.core.Animals.Animals@30f39991
! hva.core.Animals.Animals@452b3a41
! hva.core.Animals.Animals@4a574795
! hva.core.Animals.Animals@f6f4d33
! hva.core.Animals.Animals@23fc625e
! hva.core.Animals.Animals@3f99bd52
  Gestão de Animais
  1 - Visualizar todos os animais
  2 - Registar animal
