*** tests/A-01-23-M-ok.out	2024-10-11 16:08:09.351580000 +0100
--- tests/A-01-23-M-ok.outhyp	2024-10-11 22:19:13.375444906 +0100
***************
*** 16,22 ****
  3 - Vacinar animal
  4 - Mostrar vacinações
  0 - Sair
! Escolha uma opção: VACINA|V3|Tétano|0|E1
  Gestão de Vacinas
  1 - Visualizar todas as vacinas
  2 - Registar vacina
--- 16,22 ----
  3 - Vacinar animal
  4 - Mostrar vacinações
  0 - Sair
! Escolha uma opção: hva.core.Vaccine.Vaccine@30f39991
  Gestão de Vacinas
  1 - Visualizar todas as vacinas
  2 - Registar vacina
