*** tests/A-01-27B-M-ok.out	2024-10-12 19:13:01.208677000 +0100
--- tests/A-01-27B-M-ok.outhyp	2024-10-21 11:47:19.354263742 +0100
***************
*** 10,16 ****
  9 - Gestão de vacinas
  10 - Consultas
  0 - Sair
! Escolha uma opção: Ficheiro a abrir: Guardar antes de fechar? Ficheiro sem nome. Guardar ficheiro como: Menu Principal
  1 - Criar ficheiro
  2 - Abrir ficheiro
  3 - Guardar ficheiro
--- 10,16 ----
  9 - Gestão de vacinas
  10 - Consultas
  0 - Sair
! Escolha uma opção: Ficheiro a abrir: Menu Principal
  1 - Criar ficheiro
  2 - Abrir ficheiro
  3 - Guardar ficheiro
***************
*** 22,25 ****
--- 22,27 ----
  9 - Gestão de vacinas
  10 - Consultas
  0 - Sair
+ Escolha uma opção: Número inválido!
+ Escolha uma opção: Número inválido!
  Escolha uma opção: 
\ Sem newline no fim do ficheiro
