*** tests/A-04-01-M-ok.out	2024-10-18 19:27:35.000000000 +0100
--- tests/A-04-01-M-ok.outhyp	2024-10-21 23:35:46.726334776 +0100
***************
*** 18,24 ****
  5 - Plantar árvore
  6 - Mostrar árvores de habitat
  0 - Sair
! Escolha uma opção: Identificador único do habitat: Área do habitat: Gestão de Habitats
  1 - Visualizar todos os habitats
  2 - Registar habitat
  3 - Alterar área de habitat
--- 18,24 ----
  5 - Plantar árvore
  6 - Mostrar árvores de habitat
  0 - Sair
! Escolha uma opção: Gestão de Habitats
  1 - Visualizar todos os habitats
  2 - Registar habitat
  3 - Alterar área de habitat
***************
*** 26,32 ****
  5 - Plantar árvore
  6 - Mostrar árvores de habitat
  0 - Sair
! Escolha uma opção: HABITAT|AR1|Aldeia dos Macacos|3000|0
  HABITAT|AR2|Lago dos Pinguins|20|0
  Gestão de Habitats
  1 - Visualizar todos os habitats
--- 26,42 ----
  5 - Plantar árvore
  6 - Mostrar árvores de habitat
  0 - Sair
! Escolha uma opção: Número inválido!
! Escolha uma opção: Opção inválida!
! Gestão de Habitats
! 1 - Visualizar todos os habitats
! 2 - Registar habitat
! 3 - Alterar área de habitat
! 4 - Alterar influência de habitat sobre espécie
! 5 - Plantar árvore
! 6 - Mostrar árvores de habitat
! 0 - Sair
! Escolha uma opção: HABITAT|AR1|Aldeia dos Macacos|20|0
  HABITAT|AR2|Lago dos Pinguins|20|0
  Gestão de Habitats
  1 - Visualizar todos os habitats
