*** tests/A-09-08-M-ok.out	2024-10-23 19:58:54.287995577 +0100
--- tests/A-09-08-M-ok.outhyp	2024-10-24 18:49:48.780542656 +0100
***************
*** 68,81 ****
  3 - Transferir para habitat
  4 - Calcular satisfação de animal
  0 - Sair
! Escolha uma opção: Identificador único do animal: Nome do animal: Identificador único da espécie: Identificador único do habitat: Gestão de Animais
! 1 - Visualizar todos os animais
! 2 - Registar animal
! 3 - Transferir para habitat
! 4 - Calcular satisfação de animal
! 0 - Sair
! Escolha uma opção: ANIMAL|a1|patudo|C1|VOID|h1
! Gestão de Animais
  1 - Visualizar todos os animais
  2 - Registar animal
  3 - Transferir para habitat
--- 68,74 ----
  3 - Transferir para habitat
  4 - Calcular satisfação de animal
  0 - Sair
! Escolha uma opção: Identificador único do animal: Nome do animal: Identificador único da espécie: Identificador único do habitat: Nome da espécie: Gestão de Animais
  1 - Visualizar todos os animais
  2 - Registar animal
  3 - Transferir para habitat
