*** tests/A-09-03-M-ok.out	2024-10-23 19:58:54.287995577 +0100
--- tests/A-09-03-M-ok.outhyp	2024-10-24 17:07:42.583514356 +0100
***************
*** 16,34 ****
  3 - Transferir para habitat
  4 - Calcular satisfação de animal
  0 - Sair
! Escolha uma opção: Identificador único do animal: Nome do animal: Identificador único da espécie: Identificador único do habitat: Gestão de Animais
! 1 - Visualizar todos os animais
! 2 - Registar animal
! 3 - Transferir para habitat
! 4 - Calcular satisfação de animal
! 0 - Sair
! Escolha uma opção: Identificador único do animal: Nome do animal: Identificador único da espécie: Identificador único do habitat: Registar animal: Operação inválida: O animal 'a2' já existe.
! Gestão de Animais
  1 - Visualizar todos os animais
  2 - Registar animal
  3 - Transferir para habitat
  4 - Calcular satisfação de animal
  0 - Sair
  Escolha uma opção: ANIMAL|a2|animal 2|C10|VOID|h1
  Gestão de Animais
  1 - Visualizar todos os animais
--- 16,31 ----
  3 - Transferir para habitat
  4 - Calcular satisfação de animal
  0 - Sair
! Escolha uma opção: Identificador único do animal: Nome do animal: Identificador único da espécie: Identificador único do habitat: Nome da espécie: Gestão de Animais
  1 - Visualizar todos os animais
  2 - Registar animal
  3 - Transferir para habitat
  4 - Calcular satisfação de animal
  0 - Sair
+ Escolha uma opção: Número inválido!
+ Escolha uma opção: Número inválido!
+ Escolha uma opção: Número inválido!
+ Escolha uma opção: Número inválido!
  Escolha uma opção: ANIMAL|a2|animal 2|C10|VOID|h1
  Gestão de Animais
  1 - Visualizar todos os animais
