*** tests/A-09-01-M-ok.out	2024-10-18 19:27:35.000000000 +0100
--- tests/A-09-01-M-ok.outhyp	2024-10-23 21:45:06.998134873 +0100
***************
*** 16,28 ****
  3 - Transferir para habitat
  4 - Calcular satisfação de animal
  0 - Sair
! Escolha uma opção: Identificador único do animal: Nome do animal: Identificador único da espécie: Identificador único do habitat: Gestão de Animais
! 1 - Visualizar todos os animais
! 2 - Registar animal
! 3 - Transferir para habitat
! 4 - Calcular satisfação de animal
! 0 - Sair
! Escolha uma opção: ANIMAL|a1|animal1|C10|VOID|h1
  Gestão de Animais
  1 - Visualizar todos os animais
  2 - Registar animal
--- 16,22 ----
  3 - Transferir para habitat
  4 - Calcular satisfação de animal
  0 - Sair
! Escolha uma opção: Identificador único do animal: Nome do animal: Identificador único da espécie: Identificador único do habitat: Nome da espécie: Registar animal: Operação inválida: Species ID does not exist.
  Gestão de Animais
  1 - Visualizar todos os animais
  2 - Registar animal
