*** tests/A-21-10-M-ok.out	2024-10-23 19:58:54.294995495 +0100
--- tests/A-21-10-M-ok.outhyp	2024-10-25 18:31:05.956602546 +0100
***************
*** 82,90 ****
  3 - Vacinar animal
  4 - Mostrar vacinações
  0 - Sair
! Escolha uma opção: VACINA|Vac1|nome 1|1|E1
  VACINA|Vac2|nome 2|0|E2
! VACINA|Vac3|nome 3|1|E1,E2
  Gestão de Vacinas
  1 - Visualizar todas as vacinas
  2 - Registar vacina
--- 82,90 ----
  3 - Vacinar animal
  4 - Mostrar vacinações
  0 - Sair
! Escolha uma opção: VACINA|Vac1|nome 1|0|E1
  VACINA|Vac2|nome 2|0|E2
! VACINA|Vac3|nome 3|0|E1,E2
  Gestão de Vacinas
  1 - Visualizar todas as vacinas
  2 - Registar vacina
