*** testsPrivIntermedia/A-01-20-M-ok.out	2024-10-21 18:02:56.689447000 +0100
--- testsPrivIntermedia/A-01-20-M-ok.outhyp	2024-10-21 23:46:30.757159465 +0100
***************
*** 17,31 ****
  4 - Retirar responsabilidade
  5 - Calcular satisfação de funcionário
  0 - Sair
! Escolha uma opção: TRT|a24|Rohit Figueiredo
! VET|A25|Rohit Figueiredo 2
  TRT|A34|Rohit Figueiredo 3
  VET|a85|Rohit Figueiredo 4
  VET|a95|Rohit Figueiredo 5
- TRT|B24|Rohit Figueiredo 6
  TRT|d24|Rohit Figueiredo 7
  TRT|x24|Rohit Figueiredo 8
- VET|X25|Rohit Figueiredo 9
  Gestão de Funcionários
  1 - Visualizar todos os funcionários
  2 - Registar funcionário
--- 17,31 ----
  4 - Retirar responsabilidade
  5 - Calcular satisfação de funcionário
  0 - Sair
! Escolha uma opção: VET|A25|Rohit Figueiredo 2
  TRT|A34|Rohit Figueiredo 3
+ TRT|B24|Rohit Figueiredo 6
+ VET|X25|Rohit Figueiredo 9
+ TRT|a24|Rohit Figueiredo
  VET|a85|Rohit Figueiredo 4
  VET|a95|Rohit Figueiredo 5
  TRT|d24|Rohit Figueiredo 7
  TRT|x24|Rohit Figueiredo 8
  Gestão de Funcionários
  1 - Visualizar todos os funcionários
  2 - Registar funcionário
