*** testsPrivIntermedia/A-01-26-M-ok.out	2024-10-21 18:02:56.690447000 +0100
--- testsPrivIntermedia/A-01-26-M-ok.outhyp	2024-10-25 21:52:43.690964867 +0100
***************
*** 16,25 ****
  3 - Vacinar animal
  4 - Mostrar vacinações
  0 - Sair
! Escolha uma opção: VACINA|V3|Tétano|0|E1,E2,E3
! VACINA|V4|Tétano 2|0|E1,E2
! VACINA|V5|Tétano 3|0|E1,E2,E3
! VACINA|V6|Tétano|0|E2,E3
  Gestão de Vacinas
  1 - Visualizar todas as vacinas
  2 - Registar vacina
--- 16,25 ----
  3 - Vacinar animal
  4 - Mostrar vacinações
  0 - Sair
! Escolha uma opção: VACINA|V3|Tétano|0|E2,E3,E1
! VACINA|V4|Tétano 2|0|E2,E1
! VACINA|V5|Tétano 3|0|E3,E2,E1
! VACINA|V6|Tétano|0|E3,E2
  Gestão de Vacinas
  1 - Visualizar todas as vacinas
  2 - Registar vacina
