*** testsBoth/A-01-07-M-ok.out	2024-10-03 22:33:10.000000000 +0100
--- testsBoth/A-01-07-M-ok.outhyp	2024-10-11 21:29:22.377601830 +0100
***************
*** 18,24 ****
  4 - Calcular satisfação de animal
  0 - Sair
  Escolha uma opção: 1
! ANIMAL|A1|Farrusco|E1|VOID|H1
  Gestão de Animais
  1 - Visualizar todos os animais
  2 - Registar animal
--- 18,24 ----
  4 - Calcular satisfação de animal
  0 - Sair
  Escolha uma opção: 1
! hva.core.Animals.Animals@30f39991
  Gestão de Animais
  1 - Visualizar todos os animais
  2 - Registar animal
