*** tests/A-07-12-M-ok.out	2024-10-18 19:27:35.000000000 +0100
--- tests/A-07-12-M-ok.outhyp	2024-10-24 19:53:55.673476490 +0100
***************
*** 102,108 ****
  5 - Plantar árvore
  6 - Mostrar árvores de habitat
  0 - Sair
! Escolha uma opção: Identificador único do habitat: ÁRVORE|t1|t1|1|1|CADUCA|LARGARFOLHAS
  Gestão de Habitats
  1 - Visualizar todos os habitats
  2 - Registar habitat
--- 102,108 ----
  5 - Plantar árvore
  6 - Mostrar árvores de habitat
  0 - Sair
! Escolha uma opção: Identificador único do habitat: ÁRVORE|t1|t1|1|1|CADUCA|LARGAFOLHAS
  Gestão de Habitats
  1 - Visualizar todos os habitats
  2 - Registar habitat
