*** tests/A-17-08-M-ok.out	2024-10-18 19:27:35.000000000 +0100
--- tests/A-17-08-M-ok.outhyp	2024-10-22 22:35:52.614707840 +0100
***************
*** 17,26 ****
  4 - Retirar responsabilidade
  5 - Calcular satisfação de funcionário
  0 - Sair
! Escolha uma opção: TRT|T1|Trt one|H1,H2,H3
! TRT|T2|Trt two|H1,H3
! VET|V1|Vet one|C1,C3,C4
! VET|V2|Vet two
  Gestão de Funcionários
  1 - Visualizar todos os funcionários
  2 - Registar funcionário
--- 17,23 ----
  4 - Retirar responsabilidade
  5 - Calcular satisfação de funcionário
  0 - Sair
! Escolha uma opção: TRT|T1|Trt one
  Gestão de Funcionários
  1 - Visualizar todos os funcionários
  2 - Registar funcionário
