*** tests/A-21-07-M-ok.out	2024-10-18 19:27:35.000000000 +0100
--- tests/A-21-07-M-ok.outhyp	2024-10-25 02:08:54.069426856 +0100
***************
*** 28,36 ****
  3 - Vacinar animal
  4 - Mostrar vacinações
  0 - Sair
! Escolha uma opção: REGISTO-VACINA|Vac3|Vet1|E1
! REGISTO-VACINA|Vac3|Vet2|E2
! Gestão de Vacinas
  1 - Visualizar todas as vacinas
  2 - Registar vacina
  3 - Vacinar animal
--- 28,34 ----
  3 - Vacinar animal
  4 - Mostrar vacinações
  0 - Sair
! Escolha uma opção: Gestão de Vacinas
  1 - Visualizar todas as vacinas
  2 - Registar vacina
  3 - Vacinar animal
